content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
+
+This is where the Kaplan-Meier Estimator comes in. It solves both issues, since it models non-linear survival probabilities over time while accounting for subjects of the study who were unable to continue to be part of the study, but did not experience the event of interest. It does so by marking these observance as censored since it is inappropriate to say that the subject (flower) either whitered or survived. 
+
+**Calculating the survival curve manually**
+
+Back to our example. Let's imagine 10 people that all have their birthday on the same day. Their mothers, friends or spouses go and buy them a flower on that day. The probability of a flower withering the day before is probably zero, since flowershops don't sell withered flowers. On the day when these freshly cut flowers are sold and given as a gift non of them wither, but the next day three do. From this we can calculate the probability of withering and the probability of survival for that day. The probability of withering is the number of flowers withered divided by all the flowers that are at risk at that particular day:
+
+Probability of withering: 3/10 = 0.3 = 30%
+
+Probability of surviving: 7/10 = 0.7 = 70% 
+
+The probability of surviving is cumulative and so we have to multiply the probability of surviving that day with the previous probability of surviving. Since the probability of withering on the first day was 0 this means the survival probabiliy was 1 and the multiplication does not change anything for the second day. 
+On the third day another flower withered and one stopped being part of the experiment for an unknown reason (probably that pesky cat). We have to remove it from the number of flowers at risk of withering for the next day. So while on the third day there are 7 flowers at risk, since 3 whithered away the day before, on the fourth day there are only 5 at risk, since the day before one whitered away and one got censored for an unknown reason. So the probability of whithering on the fourth day is 1/5 and the probability of surviving is 4/5 multiplied by the probability of surviving the previous day, which was 0.6. Let's display all our calculations in a table:
```

Comment:
since 3 whithered --> since 3 withered
one whitered away --> one withered away
probability of whithering --> probability of withering